doc/gestor_tareas.md

```diff
@@ -0,0 +1,53 @@
+## Gestor de tareas
+
+En cuanto al task runner, sabemos que hay un montón de opciones diferentes. Primeramente, describiré los criterios utilizados para la elección. A continuación se describirán algunos de los más comunes, y luego, explicar el por qué de mi elección.
+
+### Criterios
+
+* **Integración con otras herramientas:** como ya sabemos, nuestro proyecto va a ir de la mano de Git, así que vendría muy bien elegir un task runner que se integre bien con Git.
```

Comment:
Por favor, mirad [en este archivo de errores frecuentes](https://github.com/JJ/IV-/blob/master/errores/errores-objetivo-3.md#sobre-los-criterios-para-elegir-las-herramientas) cuantos de esos criterios estáis incluyendo aquí.
En resumen, si no se puede evaluar objetiva (en el sentido de que existan o no existan) y/o numéricamente (este está más integrado en un 40%) no son criterios.